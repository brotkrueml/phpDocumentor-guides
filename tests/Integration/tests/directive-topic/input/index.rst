Directive tests
===============

..  topic:: This is the **topic**
    :class: someClass

    Lorem Ipsum Dolor

    ..  rubric:: Some Rubric

    Dolor sit!
