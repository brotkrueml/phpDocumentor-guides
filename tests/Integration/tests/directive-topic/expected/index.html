<!DOCTYPE html>
<html lang="en">
<head>
    <title>Directive tests</title>

</head>
<body>
    <div class="section" id="directive-tests">
    <h1>Directive tests</h1>

            
<div class="topic someclass">
    <p class="topic-title">This is the <strong>topic</strong></p>
    <p>Lorem Ipsum Dolor</p><div class="rubric">Some Rubric</div>
<p>Dolor sit!</p>
</div>

    </div>

</body>
</html>
